client/app/iplb/sslCertificate/iplb-ssl-certificate-edit.html

```diff
@@ -1,67 +1,50 @@
+    <oui-back-button data-title="{{ ::'iplb_ssl_add_title' | translate }}"
+                     data-on-click="ctrl.$state.go('network.iplb.detail.ssl-certificate')">
+    </oui-back-button>
 
-    <form novalidate name="ctrl.form">
-        <div class="oui-field">
-            <label for="displayName" class="oui-label" data-translate="iplb_ssl_edit_name"></label>
-            <div class="oui-field__content">
-                <div class="oui-field-control oui-field-control_4">
-                    <input type="text" id="displayName" name="displayName" class="oui-input" data-ng-model="ctrl.ssl.displayName" />
-                </div>
+    <form novalidate name="ctrl.form" data-ng-submit="ctrl.create()">
+        <oui-field label="{{ ::'iplb_ssl_edit_name' | translate }}">
+            <div class="oui-field-control oui-field-control_4">
+                <input type="text"
+                       id="displayName"
+                       name="displayName"
+                       class="oui-input"
+                       data-ng-model="ctrl.ssl.displayName">
             </div>
-        </div>
+        </oui-field>
 
-        <div class="oui-field">
-            <label for="key" class="oui-label" data-translate="iplb_ssl_edit_private_key"></label>
-            <div class="oui-field__content">
-                <div class="oui-field-control oui-field-control_5">
-                    <textarea class="oui-textarea"
-                        data-ng-class="{ 'oui-textarea_error': ctrl.form.$submitted && ctrl.form.key.$invalid }"
-                        data-ng-model="ctrl.ssl.key"
-                        id="key"
-                        name="key"
-                        required></textarea>
-                </div>
+        <oui-field label="{{ ::'iplb_ssl_edit_private_key' | translate }}">
+            <div class="oui-field-control oui-field-control_5">
+                <textarea class="oui-textarea"
+                          data-ng-model="ctrl.ssl.key"
+                          id="key"
+                          name="key"
+                          required></textarea>
             </div>
-        </div>
+        </oui-field>
 
-        <div class="oui-field">
-            <label for="certificate" class="oui-label" data-translate="iplb_ssl_edit_certificate"></label>
-            <div class="oui-field__content">
-                <div class="oui-field-control oui-field-control_5">
-                    <textarea class="oui-textarea"
-                        data-ng-class="{ 'oui-textarea_error': ctrl.form.$submitted && ctrl.form.certificate.$invalid }"
-                        data-ng-model="ctrl.ssl.certificate"
-                        id="certificate"
-                        name="certificate"
-                        required></textarea>
-                </div>
+        <oui-field label="{{ ::'iplb_ssl_edit_certificate' | translate }}">
+            <div class="oui-field-control oui-field-control_5">
```

Comment:
We should not have to add a `oui-field-control` bloc inside the `oui-field` to change the size of the field. It chains 2 blocks `oui-field-control`. The component need an attribute to change the size of the `oui-field-control`.

Reply:
yes the feature is ready
https://github.com/ovh-ux/ovh-ui-angular/commit/189f08ea18eb301a3813b2ad3f4612e418efa577